src/components/editable-title/index.tsx

```diff
@@ -0,0 +1,149 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import styled from 'styled-components';
+
+import { Color } from '../colors';
+import { CopySize } from '../copy';
+import { getSpace, SpaceSize } from '../space';
+
+export enum EditState {
+	Editable = 'Editable',
+	Editing = 'Editing'
+}
+
+export interface EditableTitleProps {
+	fontSize?: CopySize;
+	focused: boolean;
+	name: string;
+	nameState: EditState;
+	secondary?: boolean;
```

Comment:
Make this an enum, make it required

```
export EditableTitleOrder {
   Primary,
   Secondary
}
```

Reply:
Included the new enumerable to categorized the different EditableTitle component. Which I agree to be a good idea to give a much clearer idea of the component props intent :)

Reply:
I will take care now of abstracting the `EditableTitleState.Editing` and `EditableTitleState.Editable` store to the container component own state in order to solve the problem mentioned below by @tilmx.